pkg/gencosmosdb/cosmosdb/template.go

```diff
@@ -57,6 +57,7 @@ type TemplateIterator interface {
 type TemplateRawIterator interface {
 	TemplateIterator
 	NextRaw(context.Context, interface{}) error
+	ContinuationToken() string
```

Comment:
I think this might as well go into TemplateIterator, not TemplateRawIterator.  I am vaguely tempted to call this `Continuation()` to match the field name and the header name `x-ms-continuation` but I don't mind.

Reply:
I renamed `ContinuationToken` to `Continuation` and moved it to `TemplateIterator`.

The reason why I initially had it in `TemplateRawIterator` is that `templateChangeFeedIterator` (which implements `TemplateIterator`, but not `TemplateRawIterator`) sets `If-None-Match`, not `X-Ms-Continuation`.